server/services/grid_service.py

```diff
@@ -32,6 +33,15 @@ def trim_grid_to_aoi(grid_dto: GridDTO) -> geojson.FeatureCollection:
 
         # create a shapely shape from the aoi
         aoi_multi_polygon_geojson = GridService.merge_to_multi_polygon(aoi, dissolve=True)
+
```

Comment:
As discussed, testing we have a multipolygon should be responsibility of the merge_to_multi_polygon function